packages/app/src/components/inspector/GradientInspector.tsx

```diff
@@ -77,36 +73,28 @@ export default memo(function GradientInspector({
 
   const handleSubmitOpacity = useCallback(
     (opacity: number) => {
-      if (onChangeOpacity) {
-        onChangeOpacity(opacity / 100);
-      } else {
-        onChangeColor(
-          {
-            ...selectedcolor,
-            alpha: clamp(opacity / 100, 0, 1),
-          },
-          clampedSelectedStopIndex,
-        );
-      }
+      onChangeColor(
+        {
+          ...selectedcolor,
+          alpha: clamp(opacity / 100, 0, 1),
+        },
+        clampedSelectedStopIndex,
+      );
     },
-    [selectedcolor, clampedSelectedStopIndex, onChangeOpacity, onChangeColor],
+    [selectedcolor, clampedSelectedStopIndex, onChangeColor],
   );
 
   const handleNudgeOpacity = useCallback(
     (amount: number) => {
-      if (onNudgeOpacity) {
-        onNudgeOpacity(amount / 100);
-      } else {
-        onChangeColor(
-          {
-            ...selectedcolor,
-            alpha: clamp(selectedcolor.alpha + amount / 100, 0, 1),
-          },
-          clampedSelectedStopIndex,
-        );
-      }
+      onChangeColor(
```

Comment:
I don't think we want to delete the `onNudgeOpacity` case here, since that'll be important when we support modifying fills for multiple layers, all with different colors. In that case, we wouldn't want to override the entire color. Probably same with `onChangeOpacity` above

Reply:
🤔 okey!

Reply:
We should probably still put the nudge cases back in here and above